### PR TITLE
Add case about empty lines in spacing

### PR DESCRIPTION
**Pull request by arrrnas**
@vinted/ios

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -311,6 +311,27 @@ else {
 ```
 
 * There should be exactly one blank line between methods to aid in visual clarity and organization. Whitespace within methods should separate functionality, but having too many sections in a method often means you should refactor into several methods.
+ 
+* No blank lines between the last method and closing bracket of a class/struct/enum.
+
+**Preferred:**
+```swift
+class SomeClass {
+  func someFunc() {
```
Comment (ajanauskas) [inline README.markdown]: prefer 4 spaces instead of 2 :)

Comment (arrrnas) [inline README.markdown]: @ajanauskas updated

Comment (arrrnas) [inline README.markdown]: github's editor is weird, here's how it looks while editing:
screen shot 2016-08-22 at 3 19 32 pm

**Comment by ajanauskas** [comment]
👍

**Comment by MindaugasJucius** [comment]
wonderful 👍

**Comment by NerijusLas** [comment]
👍

**Comment by ajanauskas** [comment]
👏

**Comment by SimonasA** [comment]
Na pagaliau, chebra

**Comment by arrrnas** [comment]
🙇

**Pull request merged**